src/elexmodel/models/BootstrapElectionModel.py

```diff
@@ -1269,9 +1280,14 @@ def get_aggregate_prediction_intervals(
         if len(aggregate) > 1:
             aggregate_temp_column_name = "-".join(aggregate)
             all_units[aggregate_temp_column_name] = all_units[aggregate].agg("_".join, axis=1)
-            aggregate_indicator = pd.get_dummies(all_units[aggregate_temp_column_name]).values
+            dummies = pd.get_dummies(all_units[aggregate_temp_column_name])
+            aggregate_indicator = dummies.values
+            contests = dummies.columns
         else:
-            aggregate_indicator = pd.get_dummies(all_units[aggregate]).values
+            # since aggregate is of length one, we can grab the first element
+            dummies = pd.get_dummies(all_units[aggregate[0]])
+            aggregate_indicator = dummies.values
+            contests = dummies.columns
```

Comment:
Doesn't really matter but you can move this line and the one above it outside and below the `if/else` since it's the same for both cases 🤷🏻‍♀️

Reply:
yup, good call. done!